setup.py

```diff
@@ -0,0 +1,47 @@
+from setuptools import setup
+
+package_name = 'rqt_console'
+
+setup(
+    name=package_name,
+    version='1.1.1',
+    packages=find_packages(),
+    package_dir={'': 'src'},
```

Comment:
is this line needed? or the `find_packages()` above just works without this hint?